Extension/lib/webrequest.js

```diff
@@ -147,6 +147,13 @@
             return;
         }
 
+        // And we don't need this check on newer than 58 chromes anymore
+        // https://github.com/AdguardTeam/AdguardBrowserExtension/issues/572
+        if (adguard.utils.browser.isChromeBrowser() &&

```

Comment:
It had better check `webRequest.ResourceType` whether it contains `websocket` type, because this type may be added in other Chromium based (or WebExtension based) browsers later.
See https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/webRequest/ResourceType